AppCenter/AppCenter/Internals/Ingestion/MSAppCenterIngestion.m

```diff
@@ -97,9 +97,12 @@ - (NSURLRequest *)createRequest:(NSObject *)data eTag:(NSString *)__unused eTag
 }
 
 - (NSString *)obfuscateHeaderValue:(NSString *)value forKey:(NSString *)key {
-  return ([key isEqualToString:kMSHeaderAppSecretKey] || [key isEqualToString:kMSAuthorizationHeaderKey])
-             ? [MSIngestionUtil hideSecret:value]
-             : value;
+  if ([key isEqualToString:kMSAuthorizationHeaderKey]) {
+    return [NSString stringWithFormat:kMSBearerTokenHeaderFormat, @"***"];
```

Comment:
Add a util method in `MSIngestionUtil` and handle the value similar with `hideSecret`

Reply:
There is a good example implemented in this [PR](https://github.com/Microsoft/AppCenter-SDK-Apple/pull/1353/files#diff-a606764e07115ebee32c50532b58e867R44)
We probably move this to base class.